collections/_tutorials/2022-09-12-tutorial-cvae.md

```diff
@@ -83,7 +83,7 @@ $$p(y \vert x) = \int_z{p(y \vert x,z) \cdot p(z \vert x) \,dz}$$
 
 As for the variational autoencoders, the key challenge around conditional VAE is the computation of the posterior $$p(z \vert x,y)$$. Indeed, due to intractable properties, the derivation of this distribution is complicated and requires the use of approximation techniques such as variational inference.
 
-In the conditional VAE formalism, the posterior $$p(z \vert x,y)$$ is approximated by a Gaussian distribution $$q(z \vert x,y)$$ whose mean $$\mu_{post}$$ and covariance $$\sigma_{post}$$ are defined by two functions $$g(x,y)$$ and $$h(x,y)$$.
+In the conditional VAE formalism, the posterior $$p(z \vert x,y)$$ is approximated by a Gaussian distribution $$q(z \vert x,y)$$ whose mean $$\mu_{post}$$ and covariance $$\Sigma_{post}$$ are defined by two functions $$g(x,y)$$ and $$h(x,y)$$.
```

Comment:
I figure you changed to a capital sigma to stay consistent with the capital sigma used later, but I would argue with standardizing to use lower case sigmas, because that's what you use in your figures and in your tutorial on VAE. Plus you shouldn't waste time updating the figures :wink:

Unless there's some specific reason why you would want the covariance here to be a matrix? But that's not the impression I'm getting.

Reply:
Hi Nathan. I totally agree with you, this is not the proper formulation. I've worked on it and commit a new version. Just tell me if now this is fine for you !